Taller Introduccion Python - Operadores

Entrada y salida de datos

In [12]:
comunidad = "'Data Engineering Latam'"
taller = "Python: Operadores"

print("Hola", comunidad, "hoy les presentamos el taller", taller)
print(f"Hola {comunidad} hoy les presentamos el taller {taller}")
print("Hola {} hoy les presentamos el taller {}".format(comunidad, taller))

Hola 'Data Engineering Latam' hoy les presentamos el taller Python: Operadores
Hola 'Data Engineering Latam' hoy les presentamos el taller Python: Operadores
Hola 'Data Engineering Latam' hoy les presentamos el taller Python: Operadores


In [17]:
nombre = input()
edad = int(input())
print(edad + 100)

 Tulio
 30


130


Operadores Aritmeticos

SUma (+)

In [3]:
22 + 5 

27

In [9]:
entero_1 = 100 
entero_2 = 50
flotante = 5.5
suma = entero_1 + entero_2
print (suma) 

150


In [13]:
suma_2 = entero_1 + flotante 
print(suma_2)

105.5


In [14]:
print(f' La suma de {entero_1} + {entero_2} es {suma}')

 La suma de 100 + 50 es 150


In [15]:
print(f'La suma de {entero_1} + {flotante} es {suma_2}')

La suma de 100 + 5.5 es 105.5


Resta (-)

In [17]:
resta = entero_1 - entero_2 
print(resta)

50


Multiplicacion (*)

In [18]:
multiplicacion = entero_1 * flotante
print (multiplicacion)

550.0


Division (/)

In [21]:
Division = entero_1 / entero_2
print(Division)

2.0


Divion entera a la baja (//)

In [162]:
print(9//2)

4


Residuo (%)

In [22]:
print ( 30 % 6)
print (20 % 3 )

0
2


Exponente (**)

In [24]:
print ( 2 ** 3 )
print (3 ** 3)

8
27


Parentesis ()

Orden/Prioridad de Operadores aritmeticos - PEMDAS
- Parentesis, exponentes, multiplicacion-division, adicion-resta

In [26]:
Operacion1 = 100 + 50 / 5 - 4
Operacion2 = (100 + 50 / (5 - 4))
print (Operacion1) 
print (Operacion2)

106.0
150.0


Explicacion: Operacion1
100 + 10 - 4
106
Operacion2
100 + 50 / 1
100 + 50
150

Operadores de Asignacion: Asignar valores a una variable

In [151]:
numero = 10
print(numero)
print("numero")
print('numero_2')
print(type(numero))

10
numero
numero_2
<class 'int'>


Asignacion ( = )

In [10]:
comunidad = "Data Engineering Latam"
comunidad_abr = 'DEL'
numero = 100 

In [61]:
print (comunidad)
print (comunidad_abr)
print (numero)

Data Engineering Latam
DEL
100


In [55]:
suma = numero + 50
print(suma) 

150


Suma en Asignacion (+=)

In [11]:
#numero = numero + 50
numero += 50
print(numero)

150


Resta en Asignacion (-=)

In [86]:
numero_2 = 100
numero_2 -=30
print(numero_2)

70


Multiplicacion en Asignacion (*=)

In [88]:
numero_3 = 100 
numero_3 *= 5
print(numero_3)

500


Division en Asignacion (/=)

In [89]:
numero_4 = 100 
numero_4 /= 4
print(numero_4)

25.0


Operadores de comparacion

In [91]:
numero_1 = 100
numero_2 = 20
numero_3 = 5

Mayor que (>)

In [93]:
print(numero_1 > numero_2 ) # 100 > 50
print(numero_1 < numero_2 ) # 100 < 50 

True
False


Mayor o igual que (>=)

In [106]:
print(numero_1 >= numero_2)       # 100 >= 50
print(numero_1 <= numero_2)       # 100 <= 50 
print(numero_1 >= (numero_2 +50)) # 100 >= 50 + 50

True
False
True


Igualdad es (==)

Asignacion (=)

Diferente (!=)

In [103]:
print(numero_1 == numero_2 )          # 100 = 20
print(numero_1 == (numero_2 + 80 ) )  # 100 = 20 + 80 

False
True


In [104]:
operacion_c1 = numero_1 == 100

In [105]:
print (operacion_c1)

True


Ejercicio: Hallar divisores de 100

N = 100,
n = aleatorio entero

Seran divisores de N, si al dividir N por n el residuo es cero. 

In [157]:
import random
N = 100
n = random.randint(1,100)
operacion_DEL = N % n 
resultado = ( operacion_DEL == 0 )
print(resultado)
print(f"El n es: {n}")
print("El n es:", n)

False
El n es: 73
El n es: 73


Tipado Dinamico 

In [158]:
valor_dinamico = 500 
print( valor_dinamico )

500


In [159]:
valor_dinamico = "Python cambia el tipo de dato por el tipado dinamico"
print( valor_dinamico )

Python cambia el tipo de dato por el tipado dinamico


Operadores Logicos

Ejercicio

a = 2, b = 3, c = 4, d = 5

((a*c) > (b + d)) or ((d – a ) > (c – b )) and (a ==b) or (c<a)

In [6]:
a = 2
b = 3
c = 4
d = 5
((a*c) > (b + d)) or ((d - a ) > (c - b )) and (a == b) or not(c<a)

True